edi_oca/models/edi_exchange_type.py

```diff
@@ -50,7 +50,9 @@ class EDIExchangeType(models.Model):
     direction = fields.Selection(
         selection=[("input", "Input"), ("output", "Output")], required=True
     )
-    exchange_filename_pattern = fields.Char(default="{record_name}-{type.code}-{dt}")
+    exchange_filename_pattern = fields.Char(
+        default="{record_name}-{type.code}-{dt}-{seq}"
```

Comment:
IMO It shouldn't be here by default. However, we should have a help text to explain all avail variables.